networkx/algorithms/d_separation.py

```diff
@@ -312,12 +316,27 @@ def is_minimal_d_separator(G, u, v, z):
     v : node
         A node in the graph.
     z : Set of nodes
-        The set of nodes to check if it is a minimal d-separating set.
+        The set of nodes to check if it is a minimal d-separating set. A call
+        to :func:`d_separated` is called to verify that `z` is in fact a
+        d-separator is always made.
 
     Returns
     -------
     bool
-        Whether or not the `z` separating set is minimal.
+        Whether or not the set `z` is a d-separator and is also minimal.
+
+    Examples
+    --------
+    >>> G = nx.path_graph([0, 1, 2, 3], create_using=nx.DiGraph)
+    >>> G.add_node(4)
+    >>> # since {1} is the minimal d-separator, this will return False
+    >>> nx.is_minimal_d_separator(G, 0, 2, {1,3,4})
+    False
+    >>> nx.is_minimal_d_separator(G, 0, 2, {1})
+    True
```

Comment:
```suggestion
    >>> nx.is_minimal_d_separator(G, 0, 2, {1})
    True
    >>> # since {1} is the minimal d-separator, {1, 3, 4} is not minimal
    >>> nx.is_minimal_d_separator(G, 0, 2, {1, 3, 4})
    False
```
Put True result before comment, replace pronoun "this" and add spaces to the set.